src/main/java/com/yugabyte/sample/apps/AppConfig.java

```diff
@@ -46,11 +46,11 @@
   public int numWriterThreads;
 
   // The number of keys to write as a part of this workload.
-  public long numKeysToWrite;
+  public long numKeysToWrite=2000000;
```

Comment:
@gurushant-yb Proposed code change to update the defaults in `AppConfig.java` is going to affect all the workloads in Sample apps. We'll need to update the readme for all the workloads to reflect the new behavior. I believe it's going to be considerable effort testing all workloads. 

Instead of changing the default in `AppConfig`, can we just update the defaults of respective workloads (CassandraKeyValue, SqlInserts) which is used in the Platform UI. 

Do you see any drawbacks in changing just these two workloads?

Reply:
@nchandrappa : Yeah, i thought about the same,  I have observed that we have a static block in each application where we explicitly initialise 'numKeysToRead' and 'numKeysToWrite' to -1,  however it is safer side(as you recommended) to have a explicit initialization of read/write count to SqlInsert and CassandraKeyValue, done the same changes and pushed in the branch. Will be testing tomorrow.

Reply:
I am done with the testing @nchandrappa , you may proceed with the review.

Reply:
@gurushant-yb I believe this occurs when `no. of unique keys` is less than `no. of writes`.  can you check what is default value of unique keys.

For the following default values we are using
```
num_reads to 1500000 and num_writes to 2000000
```
The number of unique keys should be equal to `num_writes` or a greater value.

Reply:
@nchandrappa : Yes, when num of unique is less than num writes, then this issue occurs, Is this expected behaviour ?

Reply:
For YSQL, Yes that's the default behavior, a unique key generated is used as `PRIMARY KEY` in the SqlInserts table. As of see from the exception below.

```
[Thread-3] INFO com.yugabyte.sample.apps.SqlInserts  - Failed writing key: 5cd23d91-d32e-4cb8-95c8-36b3afdf9f17:415716
org.postgresql.util.PSQLException: ERROR: duplicate key value violates unique constraint "postgresqlkeyvalue_pkey"
```

Reply:
We should set the `--num_unique_keys` also to `2000000`.